apps/sms/views/conversation/style/message.css

```diff
@@ -89,6 +89,18 @@
   opacity: 0.5;
 }
 
+.message.sending .message-subject,
```

Comment:
Mmm, can't we just make it like this [1] @pivanov  (see only message.css changes)?

Do you any flaws in [1] that I miss?

[1] https://github.com/azasypkin/gaia/commit/23b3dc04c5c5e7f816621136bc41abc345067356#diff-ae01103c2cb0db2aec91f5c2f674a6ed